src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/emails/CourtEmailFactory.kt

```diff
@@ -76,8 +82,35 @@ object CourtEmailFactory {
         comments = booking.comments,
       )
 
-      BookingAction.RELEASED -> TODO()
-      BookingAction.TRANSFERRED -> TODO()
+      BookingAction.RELEASED -> ReleasedCourtBookingUserEmail(
```

Comment:
This isn't quite right these two need to be in another factory method for court.  This is currently the user factory method.  I will do this next.